### PR TITLE
ME-148

### PR DESCRIPTION
**Pull request by Azbagheri**
Moved the logUrl check from the BitriseClient to the main.swift file
Changed Main.swift file to exit properly even if the logs are not available

### CODE REVIEW

File: Sources/MobileTrigger/main.swift [hunk 1]
```diff
@@ -157,10 +171,6 @@ if let branch = cliMap["-b"] as? String, let workflowId = cliMap["-w"] as? Strin
         print("LOGS WERE NOT AVAILABLE - go to \(buildURL) to see log.")
```
Comment (aroswell) [inline Sources/MobileTrigger/main.swift]: Not needed any more.

File: Sources/MobileTrigger/main.swift [hunk 2]
```diff
@@ -144,8 +148,18 @@ if let branch = cliMap["-b"] as? String, let workflowId = cliMap["-w"] as? Strin
         }
```
Comment (aroswell) [inline Sources/MobileTrigger/main.swift]: in line 147 use exitCode in the exit function, and print the build message.

File: Sources/MobileTrigger/main.swift [hunk 3]
```diff
@@ -157,10 +171,6 @@ if let branch = cliMap["-b"] as? String, let workflowId = cliMap["-w"] as? Strin
         print("LOGS WERE NOT AVAILABLE - go to \(buildURL) to see log.")
     }
 
-    let msg = bitriseClient.buildStatusMessage(buildStatus)
-    print(msg)
-
-    let success = buildStatus == 1
-    let exitCode: Int32 = success ? 0 : 1
+    print(buildMessage)
```
Comment (aroswell) [inline Sources/MobileTrigger/main.swift]: move this up

File: Sources/MobileTrigger/main.swift (hunk 1 above)
Comment (Azbagheri) [reply Sources/MobileTrigger/main.swift]: removed it

File: Sources/MobileTrigger/main.swift (hunk 2 above)
Comment (Azbagheri) [reply Sources/MobileTrigger/main.swift]: Done!

File: Sources/MobileTrigger/main.swift (hunk 3 above)
Comment (Azbagheri) [reply Sources/MobileTrigger/main.swift]: Done!

**Pull request merged**